example/index.html

```diff
@@ -161,7 +161,7 @@
     <button id="fetchButton" class="btn btn-primary">Fetch Users</button>
 
   </div>
-  <script type="text/javascript" src="sdk.js"></script>
+  <script type="text/javascript" src="quiq-sdk-1.0.0.js"></script>
```

Comment:
This was downloaded from here: https://github.com/Quiq/ui-extensions/releases/tag/v1.0.0